### ISSUE TITLE
Hedge doc impossible to use for users with a screen reader

### ISSUE THREAD

**Issue by georgeowell** [hedgedoc/hedgedoc#1455]
**Description**

Two screen reader users with visual needs were unable to use our self hosted HedgeDoc and we had to move to another solution. Apologies that I wasn't able to find out what software they were using. They said "We could edit the document to some extent but we couldn't read anything properly".

I'm personally not going to have time to look into this further but thought you folks might want to know this is a problem.

**To Reproduce**

Attempt to use hedgedoc using screen reading software.

**Expected behavior**

It's possible to read and edit hedgedocs with a screen reader

**Logs**

**Additional context**

**Comment by SISheogorath** [comment]
Sounds very similar to: https://github.com/hedgedoc/hedgedoc/issues/1156

**Comment by davidmehren** [comment]
The new frontend tracks this at https://github.com/hedgedoc/react-client/issues/961.

In HedgeDoc 2, we will try to improve accessibility. As we are using external components, for example the CodeMirror editor, not everything can be influenced by us.
We will likely ask for community feedback at a later time, when the frontend is more finalized.
